skaffold/src/main/kotlin/com/google/container/tools/skaffold/SkaffoldExecutorService.kt

```diff
@@ -84,11 +91,13 @@ abstract class SkaffoldExecutorService {
  *           configuration file. If not provided, default `skaffold.yaml` used.
  * @property workingDirectory Optional, working directory where Skaffold needs to be launched.
  *           This is usually set to project working directory.
+ * @property skaffoldLabels Kubernetes style labels to apply for Skaffold deployment.
```

Comment:
super nit: but just to make it completely clear: `Kubernetes style labels to pass to Skaffold execution` ?

Reply:
done.